python/github_repo_cloner/clone_handler.py

```diff
@@ -0,0 +1,121 @@
+"""Module for CloneHandler class"""
+import subprocess
+import math
+import requests
+
+class CloneHandler:
+    """Github Clone Handler"""
+    PER_PAGE = 30.0
+    BASE_URL = 'https://api.github.com'
+
+    def __init__(self, username):
+        self.username = username
+        self.user_info = {}
+        self.repo_info = []
+        self.errors = []
+
+    @classmethod
+    def get_username(cls):
+        """Collect Github username"""
+        return input('Please enter your Github username: ')
```

Comment:
I think this probably should be in cloner.py

until it becomes repeatedly used enough to put it in an input class.

Reply:
Yep, agreed. I'll move it.